core/src/test/java/io/questdb/test/griffin/engine/groupby/TimestampSamplerFactoryTest.java

```diff
@@ -180,6 +215,16 @@ public void testUnsupportedQualifier2() {
         assertFailure(132, "unsupported interval qualifier", "21K", 130);
     }
 
+    private static void assertFindIntervalEndIndexFailure(int expectedPosition, CharSequence expectedMessage, CharSequence sampleBy, int position) {
```

Comment:
nit: can we use different numbers for "position" parameter, right now it is hardcoded to the same value of 42?